edweiss-firebase/functions/src/functions/lectures/markQuestionAsAnswered.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file markQuestionAsAnswered.ts
+ * @description Cloud function for marking a question as answered
+ * @author Adamm Alaoui
+ */
+
+// ------------------------------------------------------------
+// --------------- Import Modules & Components ----------------
+// ------------------------------------------------------------
+
+import LectureDisplay from 'model/lectures/lectureDoc';
+import { onAuthentifiedCall } from 'utils/firebase';
+import { clean, CollectionOf, getDocumentAndRef } from 'utils/firestore';
+import { fail, ok } from 'utils/status';
+
+// ------------------------------------------------------------
+// ---------   Mark Question Answered Cloud Function   --------
+// ------------------------------------------------------------
+
+export const markQuestionAsAnswered = onAuthentifiedCall(LectureDisplay.Functions.markQuestionAsAnswered, async (userId, args) => {
+    if (!args.id || args.id.length == 0)
```

Comment:
You didn't define args, you can add an interface to more readability. by exemple : 
```

interface MarkQuestionArgs {
    id: string;
    courseId: string;
    lectureId: string;
    answered: boolean;
}

```

Reply:
no problem for me, like you want